src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/tests/latest/test_servicebus_commands.py

```diff
@@ -47,6 +48,14 @@ def test_sb_namespace(self, resource_group):
         self.cmd('servicebus namespace show --resource-group {rg} --name {namespacename}',
                  checks=[self.check('sku.name', self.kwargs['sku'])])
 
+        try:
+            # Create Namespace with invalid name
+            self.cmd(
+                'servicebus namespace create --resource-group {rg} --name 1122334455 --location {loc} --tags {tags} --sku {sku}',
+                checks=[self.check('sku.name', self.kwargs['sku'])])
+        except CLIError as ex:
```

Comment:
Did you test this? I doubt this would work. self.cmd invokes the command and returns an execution result. If you want to test the exception handler method, you would need to add a unit test. You can also consider examining the execution result and checking that the correct error message is printed out when you expect a failure.

Reply:
Yes I tested, as the exception handler raise CLIError.

Reply:
Yes your response exception handler raises CLIError. 

However, the way self.cmd() is implemented, I don't believe it will actually reraise the CLIError raised by the command it executes. 

What exactly is your goal here with lines 56-57? Should you ever need a try... catch... in a test? Even if self.cmd would throw the CLIError which I doubt it will, why would you expect an exception in your test of `servicebus namespace create`. Either you expect the command to succeed or to fail. If you want to test the case that the command fails, that should be a separate test execution of the command that you expect to fail.